src/rollbar.php

```diff
@@ -480,10 +480,15 @@ protected function build_request_data() {
 
     protected function scrub_request_params($params) {
         $scrubbed = array();
+        $potential_regex_filters = array_filter($this->scrub_fields, function($field) {
+            return strpos($field, '/') === 0;
+        });
         foreach ($params as $k => $v) {
-            if (in_array($k, $this->scrub_fields)) {
-                $count = is_array($v) ? count($v) : strlen($v);
-                $scrubbed[$k] = str_repeat('*', $count);
+            if ($this->_key_should_be_scrubbed($k, $potential_regex_filters)) {
+                $scrubbed[$k] = $this->_scrub($v);
+            } elseif (is_array($v)) {
```

Comment:
yeah, that's intended. test has been amended here: cdcb2f147d7882bd7cf6683af713d1b77ca75b8e